src/modules/thirdParty/selectors.spec.ts

```diff
@@ -206,4 +216,78 @@ describe('Third Party selectors', () => {
       })
     })
   })
+
+  describe('when getting if the slot price for a third party is being loaded', () => {
+    describe('and the request is still on going', () => {
+      let state: RootState
+
+      beforeEach(() => {
+        state = {
+          ...baseState,
+          thirdParty: {
+            ...baseState.thirdParty,
+            loading: [fetchThirdPartyItemSlotPriceRequest()]
+          }
+        }
+      })
+
+      it('should return true', () => {
+        expect(isFetchingSlotPrice(state)).toEqual(true)
+      })
+    })
+  })
+
+  describe('when getting if the user is buying slots', () => {
+    describe('and the buying request is still on ongoing ', () => {
```

Comment:
Would you mind adding the negative cases for all the new tests? That is, for the `isFetchingSlotPrice` and the `isBuyingItemSlots` selectors.

Reply:
added!